cmd/nerdctl/logout.go

```diff
@@ -17,16 +17,15 @@
 package main
 
 import (
-	"fmt"
-
-	dockercliconfig "github.com/docker/cli/cli/config"
 	"github.com/spf13/cobra"
 
-	"github.com/containerd/nerdctl/v2/pkg/imgutil/dockerconfigresolver"
+	"github.com/containerd/log"
+
+	"github.com/containerd/nerdctl/v2/pkg/cmd/logout"
```

Comment:
containerd deps can be merged into one block?

Reply:
The way we currently enforce linting forimports does separate `nerdctl` deps and `containerd` deps in two sections:

https://github.com/containerd/nerdctl/blob/main/Makefile#L75-L79

https://github.com/containerd/nerdctl/blob/main/hack/lint-imports.sh#L27

We can revise this of course if this is not what we prefer, but maybe in a separate PR?